src/config/types.ts

```diff
@@ -161,6 +161,29 @@ export enum GeometryType {
   Polygon = 'polygon',
 }
 
+export enum RasterType {
+  Raster = 'raster',
+}
+
+export type HazardDataType = GeometryType | RasterType;
+
+// not including standard deviation and sum quite yet
+// because we won't be able to re-use the WMS legend
+export enum DisplayZonalStatsEnum {
```

Comment:
fair point.  I'll remove the `"Enum"`.  I agree it's not used elsewhere, so I shouldn't arbitrarily break with practice.  In case you are curious, I did it because it allows me to remember if something is a type or enum without having to move my cursor over it in VS Code.